pydoctor/templatewriter/pages/__init__.py

```diff
@@ -88,12 +88,19 @@ def part(self):
         else:
             return []
 
-    def project(self):
+    def projectURL(self):
```

Comment:
I don't understand how/if the projectURL changes are related, maybe they're not?